src/constants/index.ts

```diff
@@ -69,3 +71,8 @@ export const FUSD = new Token(
 export const BASES_TO_CHECK_TRADES_AGAINST = {
   [ChainId.FUSE]: [WFUSE, DAI, USDC, USDT, WBTC, WETH]
 }
+
+export const TOKEN_MAP = {
+  [ZERO_ADDRESS]: WFUSE_ADDRESSS,
```

Comment:
`WFUSE_ADDRESSS` is WFUSE address is checksummed? Need to keep everything in lower case to avoid errorrs